website/docs/.vitepress/sidebarConfig.ts

```diff
@@ -511,11 +511,6 @@ export const sideBarConfig: SidebarConfig = [
             activeMatch: '/data-display/bubble-chart/',
             text: 'Bubble chart',
           },
-          {
-            link: '/data-display/compact-horizontal-bar/compact-horizontal-bar',
-            activeMatch: '/data-display/compact-horizontal-bar/',
-            text: 'Compact horizontal bar bar chart',
-          },
```

Comment:
Hi there! Thanks for your request, but there is simply a wrong word order here.

The correct link is `/data-display/bar-horizontal-compact/bar-horizontal-compact`, so, please, don't delete this menu point, just change the link and activeMatch

Reply:
@ilyabrower Thanks for your quick response!

Doesn't the link you suggested already exist?

<img width="278" alt="스크린샷 2024-09-10 오후 4 29 39" src="https://github.com/user-attachments/assets/256ca930-a96e-442d-9c3a-94a5909e25af">

I wanted to remove the last item from the screenshot.

Reply:
Oh, my bad. You are right, I'll approve your request